site/guide/manage-users.qmd

```diff
@@ -35,11 +38,17 @@ To perform these next steps, you must be a group administrator or a superuser.
    
    Users now will have the option to accept the invitation through the email they receive.
 
-3. You can find pending invitations at the **Pending Invites** section.
+### Monitoring user invitations
```

Comment:
```suggestion
### Monitor user invitations
```

(We really only use gerunds for larger tasks that have discrete subtasks, i.e. task topics that have child topics.)

Reply:
Done @nrichers